src/core/rendering.js

```diff
@@ -259,10 +259,12 @@ p5.prototype.createGraphics = function(w, h, renderer) {
  * - `channels`: What color channels to store, either `RGB` or `RGBA`. The default is to match the channels in the main canvas (with alpha unless disabled with `setAttributes`.)
  * - `depth`: A boolean, whether or not to include a depth buffer. Defaults to true.
  * - `depthFormat`: The data format for depth information, either `UNSIGNED_BYTE` or `FLOAT`. The default is `UNSIGNED_BYTE` if `format` is that as well, or `FLOAT` otherwise.
- * - `antialias`: Boolean or Number, whether or not to render with antialiased edges, and if so, optionally the number of samples to use. Defaults to whether or not the main canvas is antialiased. Antialiasing is only supported when WebGL 2 is available.
+ * - `antialias`: Boolean or Number, whether or not to render with antialiased edges, and if so, optionally the number of samples to use. Defaults to whether or not the main canvas is antialiased, using a default of 2 samples if so. Antialiasing is only supported when WebGL 2 is available.
  * - `width`: The width of the texture. Defaults to matching the main canvas.
  * - `height`: The height of the texture. Defaults to matching the main canvas.
  * - `density`: The pixel density of the texture. Defaults to the pixel density of the main canvas.
+ * - `textureSmoothing`: A boolean, whether or not to interpolate between nearby pixels when reading values from the color texture. Generally, turn this on when using the texture as an image, and turn it off if reading the texture as data. Defaults to true.
```

Comment:
Thanks for adding this! Can we call it `textureFiltering` instead? I think that puts it inline with what the underlying api is called. We should also mention that true enables linear texture filtering, while false enables nearest texture filtering.